api/base_controller.go

```diff
@@ -293,6 +294,12 @@ func (c *BaseController) GetSingleObject(r *web.Request) (*web.Response, error)
 	}
 
 	stripCredentials(ctx, object)
+	displayOp := r.URL.Query().Get("display_op")
```

Comment:
How about simply `last_op`, it's a well-known term, and also whichever you choose, extract it as constant so that we don't run into yesterday's problem with hardcoded parameter names in filters which are outdated.